lib/ApiClient/AbstractApiClient.php

```diff
@@ -186,7 +186,7 @@ public function getEmbedCode($videoManagerId, $videoId, $playerDefinitionId, $em
             ]
         );
 
-        $data = \json_decode($response->getBody(), true);
+        $data = \json_decode((string) $response->getBody(), true);
```

Comment:
Do you think there is any way we can avoid using `(string)` for casting? I think on PSR-7's `StreamInterface` you can do `$response->getBody()->getContents()` but I'm not sure if Guzzle 5 already implemented this in the same way

Reply:
The reason I used this approach is because I was getting weird errors when using `getContents` on the stream. The issue is that the stream needs to be rewound to the beginning for subsequent reads. This is the approach I tried at first, and it seemed to work fine, but then suddenly during testing again I had an issue of getting an empty response. This is why I chose this simple solution, because casting to string will always return the entire response body, so it's a safer bet than handling the stream manually.